package.json

```diff
@@ -71,7 +71,11 @@
     "kbn:bootstrap": "yarn build:types && node scripts/register_git_hook",
     "spec_to_console": "node scripts/spec_to_console",
     "backport-skip-ci": "backport --prDescription \"[skip-ci]\"",
-    "cover:report": "nyc report --temp-dir target/kibana-coverage/functional --report-dir target/coverage/report --reporter=lcov && open ./target/coverage/report/lcov-report/index.html"
+    "cover:report": "nyc report --temp-dir target/kibana-coverage/functional --report-dir target/coverage/report --reporter=lcov && open ./target/coverage/report/lcov-report/index.html",
```

Comment:
thoughts on throwing these all in a `node scripts/coverage` cli?  it could give us some more flexibility/error handling

e.g. convert and send json could live in src/dev/coverage and the end to end user facing scripts at the top level

Reply:
that makes sense @jbudz , I'll follow up

Reply:
@dmlemeshko @LeeDr I think @jbudz makes a good point.  Having a single point of entry for coverage could be useful.  But, I wonder how that should work with the way coverage is launched now.  Thoughts @dmlemeshko?